### DISCUSSION TITLE
Missing copy button on 3.6.0

### DISCUSSION THREAD

**Discussion by butterford** [massCodeIO/massCode#380] [Q&A] [answered]
I might be being completely stupid but I cannot find the copy button for a snippet after installing newer version on a new laptop.

Is this a bug or am I blind, how do I copy a snippet now?

![image](https://github.com/massCodeIO/massCode/assets/5497642/e5f806a6-aeb2-42a2-a20d-798a0ec3a031)

**Comment by antonreshetov** [comment, marked as answer]
This button was only available in v1. 

Now - https://masscode.io/documentation/snippets.html#copying-snippet
- Select "Editor" > "Copy Snippet to Clipboard" from the menu bar.
- Press `Shift+Cmd+C` on macOS or `Shift+Ctrl+C` on Windows or Linux.

**Reply by butterford** [reply]
Ok, so it is me being stupid. I do wonder how that's a step forward. 

Thanks for the answer!

**Reply by diasdev** [reply]
Yeah, same here. I miss the copy button and sometimes it takes me a few seconds to realize I should use the shortcut.
